### ISSUE TITLE
Talking Head - initial face reactions

### ISSUE THREAD

**Issue by Lexx2k** [rotators/Fo1in2#19]
Bit of a weird issue. As far as I can tell, all the Fo1 reaction macros are working as intended. Yet instead of "Neutral", all characters start with "Good" mood faces.

I did some tests on Aradesh: Base reaction is 45, and thus his reaction level is 2 (Neutral). 
If I disable the `get_reaction` macro, all values are 0 and Aradesh reacts negative, as it should be.

The mood value in `start_gdialog()` isn't doing anything either, no matter the value I type in.

tl;dr - To me it seems like there is a bug in the engine and instead of showing the neutral face, it is always the smiling face?
Can be reproduced as well: Just insult Aradesh a little bit, then talk to him again... he will be smiling, no matter what the players reaction values are.

**Comment by Lexx2k** [comment]
Actually does seem to be related to the Fo1 macro. I'm guessing this needs a rewrite... ugh.

**Comment by FakelsHub** [comment]
>;Set to 1 to enable the mood argument of start_gdialog script function to be available for talking heads
;If the argument value is -1, the mood will be determined by the local variable 0 of the script (vanilla behavior)
StartGDialogFix=0

**Comment by Lexx2k** [comment]
Yeah, found that now - but it wouldn't fix the problem. The real issue is in the get_reaction macro.

I've just committed some experimental changes: https://github.com/rotators/Fo1in2/commit/468d7aabc726c13801a2ecd1c7b449d98a0f3a64
In my few tests it seems to work, though Aradesh didn't remember my good words (he did remember when I pissed him off, though).

**Comment by Lexx2k** [comment]
As far as I can tell, it works as in Fo1 now. Truth told, I didn't really do any in depth analysis of it, but if anything's still weird, we can always open a new issue ticket, so...
